src/env-var/cc-env-var-form.js

```diff
@@ -325,14 +324,14 @@ export class CcEnvVarForm extends LitElement {
 
   static get styles () {
     return [
-      defaultThemeStyles,
       linkStyles,
       // language=CSS
       css`
         :host {
-          background: #fff;
+          background-color: var(--cc-color-bg-default, #fff);
```

Comment:
no idea why I changed `background` to `background-color` but I guess this is ok anyway ?